README.md

```diff
@@ -39,32 +39,33 @@ First we create an interpreter, which requires an http4s client and
 a base url for consul:
 
 ```
+import cats.effect.IO
 import helm.http4s._
 import org.http4s.Uri.uri
 import org.http4s.client.blaze.PooledHttp1Client
 
-val client = PooledHttp1Client()
+val client = PooledHttp1Client[IO]()
```

Comment:
There has been some improvement to the client API in later versions of 0.18.  We might want to bump http4s to 0.18.11 and fix the deprecations.  (I can help if they're confusing.)